lib/features/campaigns/screens/teams_screen.dart

```diff
@@ -1,17 +1,148 @@
+    var rows = <Widget>[];
+
+    var rowCreateTeam = GestureDetector(
+      onTap: () => _beginCreateNewTeam(context),
+      child: Container(
+        padding: EdgeInsets.symmetric(horizontal: 8, vertical: 16),
+        decoration: BoxDecoration(
+          border: Border(bottom: BorderSide(width: 0.5, color: ThemeColors.textLight)),
+        ),
+        child: Stack(
+          children: [
+            Align(
+              alignment: Alignment.centerLeft,
+              child: Text(t.campaigns.team.create_team, style: theme.textTheme.bodyLarge),
+            ),
+            Align(
+              alignment: Alignment.centerRight,
+              child: Icon(Icons.chevron_right, color: theme.textTheme.bodyLarge?.color),
+            ),
+          ],
+        ),
+      ),
+    );
+    rows.add(
+      FutureBuilder(
+        future: AuthRepository().getUserInfo(),
+        builder: (context, snapshot) {
+          if (!snapshot.hasData) return SizedBox.shrink();
+
+          var userInfo = snapshot.data!;
+          if (userInfo.isCampaignManager()) {
+            return rowCreateTeam;
+          }
+          return SizedBox.shrink();
+        },
       ),
     );
+
+    return Column(children: rows);
+  }
+
+  void _beginCreateNewTeam(BuildContext context) async {
+    final theme = Theme.of(context);
+
+    var newTeamWidget = NewTeamBasicInfoWidget();
+    var newDetails = await showModalBottomSheet<NewTeamDetails>(
+      context: context,
+      builder: (context) => newTeamWidget,
+      isScrollControlled: false,
+      isDismissible: true,
+      backgroundColor: theme.colorScheme.surface,
+    );
+
+    if (context.mounted) {
+      if (newDetails != null) {
+        _showStepDivisionSelect(newDetails, context);
+      }
+    }
+  }
+
+  void _showStepDivisionSelect(NewTeamDetails newTeamDetails, BuildContext context) async {
+    final theme = Theme.of(context);
+
+    var newTeamWidget = NewTeamSelectDivisionWidget(newTeamDetails: newTeamDetails);
+    var newDetails = await showModalBottomSheet<NewTeamDetails>(
+      context: context,
+      builder: (context) => newTeamWidget,
+      isScrollControlled: false,
+      isDismissible: true,
+      backgroundColor: theme.colorScheme.surface,
+    );
+
+    if (context.mounted) {
+      if (newDetails != null) {
+        _showStepTeamLeadSelect(newDetails, context);
+      }
+    }
+  }
+
+  void _showStepTeamLeadSelect(NewTeamDetails newTeamDetails, BuildContext context) async {
+    final theme = Theme.of(context);
+    NewTeamDetails? newDetails = newTeamDetails;
+    if (!newDetails.selfJoin) {
+      var newTeamWidget = NewTeamSelectTeamLeadWidget(newTeamDetails: newTeamDetails);
+      newDetails = await showModalBottomSheet<NewTeamDetails>(
+        context: context,
+        builder: (context) => newTeamWidget,
+        isScrollControlled: false,
+        isDismissible: true,
+        backgroundColor: theme.colorScheme.surface,
+      );
+    }
+
+    if (context.mounted) {
```

Comment:
Possibly a personal preference, but these two `if` statements could also be one.